client/src/view/Home/Home.svelte

```diff
@@ -1,9 +1,40 @@
-<script>
+<script lang="ts">
+  import { onDestroy, onMount } from 'svelte'
   import Navbar from './lib/Navbar.svelte'
+  import { NavItemType } from './type/nav'
+
+  const toUpperCase = (str: string) =>
```

Comment:
This isn't really converting text to upper case, only the first letter. Something like `capitalize` or something like thath.